napari/layers/points/_tests/test_points.py

```diff
@@ -1415,33 +1415,33 @@ def test_size():
     data = 20 * np.random.random(shape)
     layer = Points(data)
     assert layer.current_size == 10
-    assert layer.size.shape == shape
-    assert np.unique(layer.size)[0] == 10
+    assert layer.size.shape == (10,)
+    assert np.unique(layer.size)[0] == [10]
```

Comment:
Why do you need a list here instead of a plain value?

```suggestion
    assert np.unique(layer.size)[0] == 10
```